The two tests are identical. They should not generate any errors
- Test with the two different after() methods

mockgoose.reset() seems to break unique index of mongoose
